src/components/connections.vue

```diff
@@ -31,33 +48,47 @@ export default {
   .box
     h1
       span Connections
-      button(@click="isNewConnectionFormVisible = !isNewConnectionFormVisible").round.add {{ isNewConnectionFormVisible ? '-' : '+' }}
-    form(v-if="isNewConnectionFormVisible") 
+      button.round.add(@click='isNewConnectionFormVisible = !isNewConnectionFormVisible') {{ isNewConnectionFormVisible ? "-" : "+" }}
+    form(v-if='isNewConnectionFormVisible')
       .form-item
         p Connection name:
-        input(type="text" v-model="name")
+        input(type='text', v-model='name')
       .form-item
         p Connection string:
-        input(type="text" v-model="connectionString")
-      button(type="button" @click="doConnect") Connect
+        input(type='text', v-model='connectionString')
+      button(type='button', @click='doConnect') Connect
   .box
-     h1 Databases
-     .database(v-for="database in databases" @click="setDatabase(database.name)")
-       p {{database.name}}
+    h1 Databases
+    .box-item(
+      v-for='database in databases',
+      @click='setSelectedDBorCollection("db", database.name)',
+      :class='{ selected: database.name === databaseName ? true : false }'
```

Comment:
`database.name == databaseName` is enough here, you don't need a ternary. the same for the collections below.